node/src/transaction_pool/candidate/transaction_pool_candidate_state.rs

```diff
@@ -15,10 +16,17 @@ use crate::p2p::PeerId;
 static EMPTY_PEER_TX_CANDIDATES: BTreeMap<TransactionHash, TransactionPoolCandidateState> =
     BTreeMap::new();
 
+type NextBatch = Option<(
+    PeerId,
+    Vec<TransactionWithHash>,
+    Option<P2pNetworkPubsubMessageCacheId>,
```

Comment:
As mentioned above, this should have the actual source. Also I would not make this an `Option` alias, lets define a proper struct here and have `get_batch_to_verify` return `Option`